service_mesh/v2x/ossm-create-smcp.adoc

```diff
@@ -28,8 +28,7 @@ include::modules/ossm-validate-smcp-kiali.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 == Additional resources
-
-{SMProductName} supports multiple independent control planes within the cluster. You can create reusable configurations with `ServiceMeshControlPlane` profiles. For more information, see xref:../../service_mesh/v2x/ossm-profiles-users.adoc#ossm-control-plane-profiles_ossm-profiles-users[Creating control plane profiles].
+* xref:../../service_mesh/v2x/ossm-profiles-users.adoc#ossm-control-plane-profiles_ossm-profiles-users[Creating control plane profiles].
```

Comment:
You could remove the full stop from this bullet point, since it is not a full sentence.